.github/workflows/container-image.yaml

```diff
@@ -30,14 +21,6 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v3
 
-      - name: Run e2e tests against the example
```

Comment:
I'd comment this out rather than removing them. We indeed need these e2e test to make sure the whole integration works.

Reply:
Closing the comment as I see you turned e2e tests into go code :).